metadata/nz_building_outlines.xml

```diff
@@ -312,7 +306,36 @@ If you adapt this work in any way or include it in a collection, and publish, di
       <gmd:lineage>
         <gmd:LI_Lineage>
           <gmd:statement>
-            <gco:CharacterString>Building outlines have been identified as a dataset of national importance. In 2016 LINZ conducted a pilot project to capture building outlines over three regions of New Zealand (Canterbury, Hawke’s Bay and Waikato). A survey was sent out to assess users’ opinion and determine the usefulness and suitability of the data for their purposes. The majority of the respondents (90%) to the survey agreed that the data is useful for their organisation. Many commented that the data is better than existing data, it is fit for purpose, and has become invaluable when enriched with other datasets. LINZ will continue to procure building outlines aligning with aerial imagery capture.</gco:CharacterString>
+            <gco:CharacterString>
+**Source imagery**
+Resolution of the aerial imagery used for building outline capture ranges from 50cm pixel resolution (0.5m GSD) to 7.5cm pixel resolution (0.075m GSD).  The source imagery is linked via the external_source_id attribute of building outlines to the imagery_survey_id of NZ Imagery Surveys. Using this link, additional attributes can be connected to building outlines - for example the dates that the imagery was captured and the accuracy / ground sample distance of the imagery used.
+
+The NZ Imagery Surveys data dictionary is here: https://nz-imagery-surveys.readthedocs.io/en/latest/index.html
+
+**Accuracy Specification:**
+The Building Outlines contain spatial detail reflective of the visible characteristics of building outlines as seen from the source aerial imagery. The building outlines were created by an both an automated and manual method of feature extraction. Outlines are captured in full where they are partially occluded by vegetation or require additional viewer interpretation, and squared off at 90 degrees if the corner angles are between 80-100 degrees.
```

Comment:
The building outlines were created by **an both an automated and manual** method of feature extraction. 
too many `an` :smile_cat: